hibike/unit_tests/hibike_process_tests.py

```diff
@@ -0,0 +1,65 @@
+"""
+Unit tests for functions in hibike_process.
+"""
+import unittest
+import time
+import serial
+from spawn_virtual_devices import spawn_device, get_virtual_ports
+from hibike_process import identify_smart_sensors
+import hibike_message as hm
+
+class IdentifySmartSensorsTests(unittest.TestCase):
+    """
+    Tests for `identify_smart_sensors`.
+    """
+    VIRTUAL_DEVICE_TYPES = ["LimitSwitch", "YogiBear", "YogiBear",
```

Comment:
Will this ever be used / synced with something else? If so, it should be in a central file that all things are importing.